### PR TITLE
Moving list to the new TodoList component and adding the Add ToDo for…

### PR DESCRIPTION
**Pull request by AigulY**
Lesson_1_2

### CODE REVIEW

**Comment by JoHeyJo** [comment]
I want to make sure that you are aware that you are using different indentations throughout your files. 
`App` has a 2-space indentation while `TodoList` and `AddTodoForm` have a 4-space indentation.  While this isn't a problem, reading between files can be confusing, especially as the code base grows. I suggest picking either 2 or 4. Now, there is no set rule for indentation in JavaScript (not like Python, which is a strict 4 spaces) but I will say that the broader community has landed on 2 spaces. I  suggest going with whatever works for you for the time being. (just keep in mind that you may need to switch to 2 spaces if you continue working with JavaScript)

Also, nicely formatted files!

File: src/AddTodoForm.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+const AddTodoForm = () => {
+    return (
+        <div>
+            <form>
```
Comment (JoHeyJo) [inline src/AddTodoForm.js:6]: Nice form!

File: src/TodoList.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+function TodoList() {
+    const todoListData = [
```
Comment (JoHeyJo) [inline src/TodoList.js:4]: I really like your variable naming! Very descriptive. This is a great practice to always consider. Have your variables be as descriptive as possible. Keep in mind that as your code base and data input grows you're going to have a lot of "data". It's not necessary to include "data" in the name when you already have a very descriptive variable name. 

Rule of thumb: "Try not to have generic names for variables when possible"

**Comment by JoHeyJo** [comment]
Great job! A few minor suggestions in the comments. Then merge it!

**Comment by AigulY** [comment]
> I want to make sure that you are aware that you are using different indentations throughout your files. `App` has a 2-space indentation while `TodoList` and `AddTodoForm` have a 4-space indentation. While this isn't a problem, reading between files can be confusing, especially as the code base grows. I suggest picking either 2 or 4. Now, there is no set rule for indentation in JavaScript (not like Python, which is a strict 4 spaces) but I will say that the broader community has landed on 2 spaces. I suggest going with whatever works for you for the time being. (just keep in mind that you may need to switch to 2 spaces if you continue working with JavaScript)
> 
> Also, nicely formatted files!

Good night!

yes, I am aware. I also thought it is strange )) As you recommended I searched and find out that used Shift+Alt +F that helps to automatically correct my indentations and it gave me this indentations, that I decided not to touch as it was recommended by the system. I will correct it. Thanks a lot! )

**Pull request merged**